### PR TITLE
refactor: migrate transform panels to new style engine

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio/issues/1536

Switched translate, scale, rotate and skew panels to new style engine. Simplified style updates.

image
image
image
image

### CODE REVIEW

File: apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx [hunk 1]
```diff
@@ -266,34 +177,29 @@ export const ScalePanelContent = (props: TransformPanelProps) => {
             description={propertySyntaxes.scaleZ}
           />
           <CssValueInput
-            key="scaleZ"
             styleSource="local"
-            property={fakeProperty}
+            property={property}
```
Comment (JayaKrishnaNamburu) [inline apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx:181]: Screenshot 2024-09-19 at 11 12 51 AM


sclae-none-bug

We need to set the `fakeProperty` here, or else it allowing user to set keywords. But when used in `scaleX` and `scaleY` keywords can't be set of just one axis

**Review by JayaKrishnaNamburu — APPROVED** [review]
Looking good, and working as expected. Just a small change for `property` update for scale.

File: apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx (hunk 1 above)
Comment (TrySound) [reply apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx:181]: Hm, this is not a keyword. It's some weird unit in there.

Comment (JayaKrishnaNamburu) [reply apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx:181]: It comes from here 
https://developer.mozilla.org/en-US/docs/Web/CSS/scale#syntax

We are setting property as `scale` directly and so the `CSSValueInput` is picking the keywords available for the property directly. But in reality we don't allow to set just one value for the scale from UI right. So used `opacity` before which has the same syntax for individual values and no keywords like `none` as such.
https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#syntax

Comment (TrySound) [reply apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx:181]: I mean we do not pass keywords list of css input. And you are pointing to unit select. For some reason it shows keyword, but it shouldn't.

Comment (JayaKrishnaNamburu) [reply apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx:181]: It comes from here
https://github.com/webstudio-is/webstudio/blob/main/apps/builder/app/builder/features/style-panel/shared/css-value-input/unit-select.tsx#L50

`buildOptions` is taking keywords from the auto-generated properties list to show under it. So even if the passed `keywords` list is empty. It picks from the `UnitSelector`.

Comment (TrySound) [reply apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx:181]: Killed it

**Pull request merged**